inst/mappings/mapping_ECEMF.csv

```diff
@@ -311,6 +311,10 @@ Energy Service|Transportation|Passenger|LDV;Gpkm/yr;;;;;;;
 Energy Service|Transportation|Passenger|Rail;Gpkm/yr;;;;;;;
 Energy Service|Transportation|Passenger|Truck;Gpkm/yr;;;;;;;
 Energy System Cost;billion EUR_2020/yr;Energy system costs;billion US$2017/yr;0.9502;;;;R
```

Comment:
I guess we should take this one out then, right? 
with the new definition of "Energy System Cost|Supply", our total clearly doesn't match anymore the "Energy System Cost" variable that includes demand